event-grid/filter-events-preview/filter-events-preview.ps1

```diff
@@ -0,0 +1,34 @@
+﻿# You must have the latest version of the Event Grid PowerShell module.
+# To install:
+# Install-Module -Name AzureRM.EventGrid -AllowPrerelease -Force -Repository PSGallery
+
+# Provide an endpoint for handling the events.
+$myEndpoint = "<endpoint URL>"
+
+# Provide the name of the custom topic to create
+$topicName = "<your-topic-name>"
+
+# Provide the name of the resource group to contain the custom topic
+$myResourceGroup= "<resource-group>"
+
+# Select the Azure subscription that contains the resource group.
+Set-AzureRmContext -Subscription "Contoso Subscription"
```

Comment:
Other user-supplied values are replacable. This should be replaced as well, or should rely on the user having already conducted a login.